providers/flagd/pkg/provider.go

```diff
@@ -90,8 +100,7 @@ func WithSocketPath(socketPath string) ProviderOption {
 // FromEnv sets the provider configuration from environment variables: FLAGD_HOST, FLAGD_PORT, FLAGD_SERVICE_PROVIDER, FLAGD_SERVER_CERT_PATH & FLAGD_CACHING_DISABLED
 func FromEnv() ProviderOption {
 	return func(p *Provider) {
-
-		portS := os.Getenv("FLAGD_PORT")
+		portS := os.Getenv(flagdPortEnvironmentVariableName)
```

Comment:
Why is the S uppercase?

Reply:
I'm not sure tbh, I hadn't noticed the S. Happy to change it

Reply:
S is for string in this instance, portString

Reply:
Okay, thanks. Must be a Go thing 😄